duffy/worker.py

```diff
@@ -0,0 +1,94 @@
+'''
+<TODO>
+This is a [WIP] python file to update worker from v1.
+'''
+
+from config import ProdConfig
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import json
+import subprocess
+
+import beanstalkc
+bs_obj = beanstalkc.Connection(host='127.0.0.1', parse_yaml=False)
+bs_obj.watch('requests')
+bs_obj.ignore('default')
+
+app = Flask(__name__)
+app.config['DEBUG'] = ProdConfig.DEBUG
+app.config['DEBUG_TB_ENABLED'] = ProdConfig.DEBUG_TB_ENABLED
+app.config['ENV'] = ProdConfig.ENV
+app.config['SQLALCHEMY_DATABASE_URI'] = ProdConfig.SQLALCHEMY_DATABASE_URI
+
+db_obj = SQLAlchemy(app)
+
+def connection_check():
+    '''
+    This function checks and establishes sqllite Connection.
+    '''
+    try:
+        dummy_data = stock.query.first()
+    except:
+        global db_obj
+        global app
```

Comment:
If you use the duffy app directly to get the DB, you can do away with the connection check.